lib/components/Application.js

```diff
@@ -39,26 +50,39 @@ export default class Application extends Component {
     this.setState({ draftMessage: '' });
   }
 
+  updateState(e) {
+    this.setState( { draftMessage: e.target.value });
+  }
+
   render() {
     const { user, messages, draftMessage } = this.state;
 
     return (
       <div className="Application">
-        {user ? <p>Hello {user.displayName}</p> : <button onClick={() => signIn()}>Sign In</button> }
```

Comment:
good to get rid of this line for now, maybe we can add a new component for it even later